nav2_collision_monitor/src/collision_detector_node.cpp

```diff
@@ -82,7 +82,7 @@ CollisionDetector::on_activate(const rclcpp_lifecycle::State & /*state*/)
   }
 
   // Creating timer
-  timer_ = this->create_timer(
+  timer_ = this->create_wall_timer(
```

Comment:
why?

Reply:
There is not node method in humble called create_timer.. that is only for jazzy
There is a rclcpp::create_timer but the original in humble is this->create_wall_timer()